core/domain/user_jobs_one_off.py

```diff
@@ -254,3 +255,38 @@ def map(item):
     @staticmethod
     def reduce(key, stringified_values):
         pass
+
+
+class UserLastExplorationActivityOneOffJob(jobs.BaseMapReduceJobManager):
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [user_models.UserSettingsModel]
+
+    @staticmethod
+    def map(user_model):
+        user_id = user_model.id
+        contributions = user_models.UserContributionsModel.get(user_id)
+
+        created_explorations = exp_services.get_multiple_explorations_by_id(
+            contributions.created_exploration_ids)
+        if created_explorations:
+            user_model.last_created_an_exploration = max(
+                [model.created_on for model in created_explorations.values()])
+
+        user_commits = (
+            exp_models.ExplorationCommitLogEntryModel.query(
+                exp_models.ExplorationCommitLogEntryModel.user_id == user_id).
+            order(-exp_models.ExplorationCommitLogEntryModel.created_on).
+            fetch())
+
+        for commit in user_commits:
+            if commit.commit_type != 'create':
```

Comment:
Rahim hasn't replied. I think he's busy, so I'm going to say, let's also count creation events as edits for now, and drop this check. That way, at least we all have a common understanding of how things work. If this changes subsequently we'll re-introduce the condition, but I think we should prioritize getting this merged.